LOADERS/REMOTE_FILE_SYSTEM/REMOTE_FILE/REMOTE_FILE.py

```diff
@@ -0,0 +1,103 @@
+from urllib.parse import urlparse
+
+from flojoy import flojoy, Image, DataFrame, Grayscale, TextBlob
+from typing import Literal, Optional
+import numpy as np
+from PIL import Image as PIL_Image
+import pandas as pd
+
+import requests
+from io import BytesIO
+
+
+def get_file_url(url: str):
+    parse_result = urlparse(url)
+    if not all([parse_result.scheme, parse_result.netloc]):
+        raise ValueError(f"'{url}' is not a valid URL.")
+    return url
+
+
+@flojoy(
+    deps={
+        "xlrd": "2.0.1",
+        "lxml": "4.9.2",
+        "openpyxl": "3.0.10",
+        "scikit-image": "0.21.0",
+    }
+)
+def REMOTE_FILE(
+    file_url: str = None,
+    default: Optional[TextBlob] = None,
+    file_type: Literal["Image", "Grayscale", "JSON", "CSV", "Excel", "XML"] = "Image",
+) -> Image | DataFrame:
+    """The REMOTE_FILE node loads a remote file of a different type and converts it to a DataContainer class.
```

Comment:
I would specify that this only supports HTTP file URLs and that `gcp://`, `s3://` and other bucket-like URIs are not supported